test/exec-util.ts

```diff
@@ -7,19 +8,26 @@ import { regEx } from '../lib/util/regex';
 
 type CallOptions = ExecOptions | null | undefined;
 
+// TODO: rename
 export type ExecResult = { stdout: string; stderr: string } | Error;
 
 // TODO: fix type #7154
 export type ExecMock = jest.Mock<typeof _exec>;
 export const exec: ExecMock = _exec as any;
 
+export type SpawnMock = jest.MockedFunction<typeof _spawn>;
+export const spawn = _spawn as SpawnMock;
```

Comment:
Do we need this, i don't see any reference to it. i think `ecec` can now also be removed?

Reply:
both are used, `exec` is a mock of the wrapper promisifing function we added in this PR (will be renamed).

and spawn is used as a mock for `child_process.spawn`
and is used mainly in here -

https://github.com/renovatebot/renovate/blob/d9c46295911022e023bc28a47a2053e1452297ee/lib/util/exec/common.spec.ts#L1-L3